include/verilated_vpi.cpp

```diff
@@ -2384,122 +2422,210 @@ static void vl_strprintf(std::string& buffer, char const* fmt, ...) {
     va_end(args);
 }
 
-void vl_get_value(const VerilatedVar* varp, void* varDatap, p_vpi_value valuep,
-                  const char* fullname) {
+template<typename T>
+struct VarAccessInfo final {
+    size_t bitOffset;
+    size_t wordOffset;
+    T mask_lo, mask_hi;
+    T* ptr;
```

Comment:
```suggestion
    size_t m_bitOffset;
    size_t m_wordOffset;
    T m_maskLo;
    T m_maskHi;
    T* m_datap;
```

And comment each please.